spec/models/rental_spec.rb

```diff
@@ -223,4 +223,31 @@
     rent = create :mock_rental, item_type: create(:item_type, name: 'Test 220', base_fee: 200, fee_per_day: 20)
     expect(FinancialTransaction.where(rental: rent).map(&:amount)).to eq([220])
   end
+
+  describe '#delete_reservation' do
+    context 'error thrown' do
+      it 'logs the error and returns false' do
+        r = build(:rental)
+        r.create_reservation
```

Comment:
Are the two lines needed, by any chance? Or are the callbacks not firing? What I am suggesting is something along the lines of...

```Ruby
it 'logs the error and returns false' do
  allow(Inventory).to receive(:delete_reservation).and_raise(InventoryExceptions::AuthError)
  expect(create(:rental).delete_reservation).to be false
end
```